### PR TITLE
Initial implementation for WeekDataAccess. Initial implementation for…

### PR DESCRIPTION
**Pull request by DanielLivshits**
… the Data Access versions of the entities.

### CODE REVIEW

**Review by POPPININ — CHANGES_REQUESTED** [review]
Please finish the WeekDataAccess class and add appropriate documentation to all classes proposed.

**Review by Fard-Faru — CHANGES_REQUESTED** [review]
Hi Daniel, I noticed you are missing Documentation and test cases.

**Review by POPPININ — APPROVED** [review]
Looks good.

File: src/test/java/data_access/DataAccessTaskTest.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    @Test
+    void testSetCompleted() {
+        DataAccessTask study = new DataAccessTask("study a lot");
+        study.setCompleted(true);
+        assertTrue(study.getCompleted());
+    }
+
+    @Test
+    void testToString() {
+        DataAccessTask sleep = new DataAccessTask("get some sleep");
```
Comment (Fard-Faru) [inline src/test/java/data_access/DataAccessTaskTest.java:36]: We all need some haha

**Review by Fard-Faru — APPROVED** [review]
Looks, Good Daniel!

**Review by scissorkhizer — APPROVED** [review]
Thank you for setting up the initial implementation for the WeekDataAccess and the corresponding classes. The tests have good breadth and depth based on the implementation thus far!

**Pull request merged**